File: Assignments/Assignment 28/C Codes/program28_2.c
/*
Input : iRow = 4        iCol = 4

Output : 
        A   B   C   D
        a   b   c   d
        A   B   C   D
        a   b   c   d
*/

#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0;
    int j = 0;
    int iCapital = 0;
    int iSmall = 0;

    for(i = 1; i <= iRow; i++)
    {
        iCapital = 65;
        iSmall = 97;

        for(j = 1; j <= iCol; j++)
        {
            if(i%2 != 0)
            {
                printf("%c\t", iCapital);
                iCapital++;
            }
            else
            {
                printf("%c\t", iSmall);
                iSmall++;
            }
        }
        printf("\n");
    }

}

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter number of rows :\n");
    scanf("%d", &iValue1);

    printf("Enter number of columns :\n");
    scanf("%d", &iValue2);

    Pattern(iValue1, iValue2);

    return 0;
}

File: Assignments/Assignment 28/Java Codes/program28_2.java
/*
Input : iRow = 4        iCol = 4

Output : 
        A   B   C   D
        a   b   c   d
        A   B   C   D
        a   b   c   d
*/

import java.util.*;

class Pattern {
    public void Display(int iRow, int iCol) {
        int i = 0;
        int j = 0;
        char cCapital = '\0';
        char cSmall = '\0';

        for (i = 1; i <= iRow; i++) {
            cCapital = 65;
            cSmall = 97;
            for (j = 1; j <= iCol; j++) {
                if (i % 2 != 0) {
                    System.out.print(cCapital + "\t");
                    cCapital++;
                } else {
                    System.out.print(cSmall + "\t");
                    cSmall++;
                }
            }
            System.out.println("");
        }
    }
}

class program28_2 {
    public static void main(String A[]) {
        {
            Scanner sobj = new Scanner(System.in);
            int iValue1 = 0;
            int iValue2 = 0;

            System.out.println("Enter number of rows :");
            iValue1 = sobj.nextInt();

            System.out.println("Enter number of columns :");
            iValue2 = sobj.nextInt();

            Pattern pobj = new Pattern();

            pobj.Display(iValue1, iValue2);
        }
    }
}